### ISSUE TITLE
Facing issues while trying to run the web server on my remote instance

### ISSUE THREAD

**Issue by SwapnaliDive20** [bcgov/von-network#163]
Hello 
I am trying to setup von-network web server on my remote instance. 

I did the following steps 

1. git clone https://github.com/bcgov/von-network.git
2. cd von-network
3. virtualenv --python=python3.6 venv
4. source venv/bin/activate
5. pip install -r server/requirements.txt

On doing the above step I am facing the following issues. 

ERROR: Could not find a version that satisfies the requirement Charm-Crypto (from indy-anoncreds-dev) (from versions: 0.43)
ERROR: No matching distribution found for Charm-Crypto

This error occured while installing indy-anoncreds-dev. 

we are using python3.6

Any help will be appreciated.

**Comment by WadeBarnes** [comment]
@SwapnaliDive20, Are you using `von-network` for the remote ledger instance, or are you just wanting to use the ledger browser portion of `von-network` for your remote ledger instance?

**Comment by SwapnaliDive20** [comment]
@WadeBarnes Yes I am using von-network and I want to use the ledger browser portion of von-network for my remote ledger instance

**Comment by WadeBarnes** [comment]
@SwapnaliDive20, The easiest way to accomplish this is to publish the `pool_transactions_genesis` file for you're network on a URL, and then run the `von-network` ledger browser using docker.

Example of connecting the `von-network` ledger browser to Sovrin StagingNet:
```bash
./manage build
GENESIS_URL=https://raw.githubusercontent.com/sovrin-foundation/sovrin/master/sovrin/pool_transactions_sandbox_genesis ./manage start-web
```
